build-tarball.sh

```diff
@@ -29,6 +29,18 @@ cp $dir/recovery.img $wDir/partitions
 cp -r "$device_dir/ubuntu-overlay/*" $wDir/ || true
 
 mkdir -p $wDir/system/var/lib/lxc/android/
+
+## SPARSE FILE TRANSLATION
+# Needed with Halium-7.1 builds mostly
+isSparse=$(file -b0 $dir/system.img)
+if [[ $isSparse == "Android sparse image"* ]]; then
+    echo "Converting sparse image to image"
+    mv $dir/system.img $dir/system.sparse.img
+    simg2img $dir/system.sparse.img $dir/system.img
+    e2fsck -fy $dir/system.img >/dev/null
```

Comment:
Do you mean `&>/dev/null`? This will only send stdout, not stderr, to null.

Reply:
Huh what do I mean I have to think about it ^^

Reply:
I think better leave errors in the log. That could show potential problems better.